provers/stark/src/table.rs

```diff
@@ -15,7 +15,7 @@ pub struct Table<F: IsFFTField> {
 impl<F: IsFFTField> Table<F> {
     /// Crates a new Table instance from a one-dimensional array in row major order
     /// and the intended width of the table.
-    pub fn new(data: &[FieldElement<F>], width: usize) -> Self {
```

Comment:
Is this needed ?

Reply:
I think it is better for the `data` vector to be consumed once the `Table` struct is initialized with it. I can't think of any use case where we want to preserve the `data` vector by itself, what do you think?